modules/network/src/main/java/org/apache/ignite/internal/network/NodeFinderFactory.java

```diff
@@ -51,10 +53,22 @@ public static NodeFinder createNodeFinder(NodeFinderView nodeFinderConfiguration
                 return Arrays.stream(nodeFinderConfiguration.netClusterNodes())
                         .map(NetworkAddress::from)
                         .collect(collectingAndThen(toUnmodifiableList(), StaticNodeFinder::new));
+            case MULTICAST:
+                MulticastView multicastConfig = nodeFinderConfiguration.multicast();
 
+                MulticastNodeFinder multicastNodeFinder = new MulticastNodeFinder(
+                        multicastConfig.group(),
+                        multicastConfig.port(),
+                        multicastConfig.discoveryResultWaitMillis(),
+                        multicastConfig.ttl(),
+                        localAddress
+                );
+
+                multicastNodeFinder.start();
```

Comment:
This method is just about creating a `NodeFinder`. Let's add `start()` method on `NodeFinder` and invoke it not here, but in the calling method after the `NodeFinder` has been created